tools/foxglove/cereal_to_mcap.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
```

Comment:
this file isn't actually executable

Reply:
interesting, it has the execute bit set on a checkout for me and if i remove the execute bit it shows up as a diff in git